src/Umbraco.Web.UI/config/umbracoSettings.config

```diff
@@ -54,7 +54,7 @@
     <disallowedUploadFiles>ashx,aspx,ascx,config,cshtml,vbhtml,asmx,air,axd,swf,xml,xhtml,html,htm,php,htaccess</disallowedUploadFiles>
 
     <!-- You can specify your own background image for the login screen here. The image will automatically get an overlay to match back office colors - this path is relative to the ~/umbraco path. The default location is: /umbraco/assets/img/installer.jpg -->
-    <loginBackgroundImage>assets/img/installer.jpg</loginBackgroundImage>
+    <loginBackgroundImage>assets/img/login.jpg</loginBackgroundImage>
```

Comment:
@nielslyngsoe , I think you also have to update the `umbracoSettings.Release.Config`, otherwise it still uses the `installer.jpg` in production

Reply:
@bergmania thanks, Its corrected now.